File: Assignment25/program2.c
/////////////////////////////////////////////////////////////////// 
//
// File name : program2.c
// Description : Accept string and convert to upper case
// Author : Adinath Santosh Pawar
// Date : 6/6/25
// 
///////////////////////////////////////////////////////////////////

#include<stdio.h>

void struprX(char *str)
{
   while(*str != '\0')
   {
        if( (*str >= 'a') && (*str <= 'z') ) 
        {
           *str = *str - 32; 
        }
        str++;
   } 
   
}

int main()
{
    char arr[20] = {'\0'};

    printf("enter String \n");
    scanf("%[^'\n]s",arr); 

    struprX(arr);
    printf("Modified string is %s \n ",arr);

    return 0;
}

File: Assignment45/program2.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program2.java
// Description : reverse content of each row of matrix
// Author : Adinath Santosh Pawar
// Date : 2/8/25
// 
///////////////////////////////////////////////////////////////////

class Matrix
{
    private int Arr[][];

    public Matrix(int A, int B)
    {
        Arr = new int[A][B];
    }
        
    public void Accept()
    {
        System.out.println("Enter the elements of matrix ");

        Scanner sobj = new Scanner(System.in);

        int i = 0, j = 0;
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
    }

    public void ReverseRow(int iRow,int iCol)
    {
        int i = 0, j = 0;
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < (Arr[i].length)/2; j++)
            {
                int temp = Arr[i][j];
                Arr[i][j] = Arr[i][iCol-j-1];
                Arr[i][iCol-j-1] = temp;
            }
        }
    }

    public void Display()
    {
        System.out.println("The elements of matrix are ");

        int i = 0, j = 0;
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                System.out.print(Arr[i][j] + " ");
            }
            System.out.println();
        }
    }

}
class program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet = 0;

        System.out.println("Enter number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter number of columns :");
        int iCol = sobj.nextInt();

        Matrix mobj = new Matrix(iRow,iCol);

        mobj.Accept();
        mobj.ReverseRow(iRow,iCol);
        mobj.Display();
    }
}